src/app/tests/suites/certification/Test_TC_ACFREMON_2_1.yaml

```diff
@@ -68,9 +68,10 @@ tests:
       command: "readAttribute"
       attribute: "ChangeIndication"
       response:
-          value: 1
           constraints:
               type: enum8
+              minValue: 0
```

Comment:
Test plan seems to say "Value has to be 0 or 2". Min/max of 0/2 allows the value "1" as valid and that does not seem to be what the test plan says.

Reply:
It looks like the type would allow 0,1 or 2 ... where 0 is OK, 1 is Warning and 2 is Critical. Test plan says only OK and Critical are allowed though ...

This likely needs review by someone working on the spec/test plan side of the cluster.

Reply:
I see the `1: If ACFREMON.S.F01(Warning)` and pics guards here, missed that until I expanded the test. Seems ok.